docusaurus/docs/reference/ha/migrating.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: Migrating
+sidebar_position: 30
+---
+
+# Migrating Controllers
+
+A controller can be moved from standalone mode to HA mode. It can also be returned
+from HA mode back to standalone mode.
+
+## Standalone to HA
+
+### Requirements
+First, ensure that the controller's certificates and configuration meet the requirements
+in [Bootstrapping](./bootstrapping.md).
```

Comment:
shouldn't this be pointing to certificates?

Reply:
ooooh and now i caught the "and configuration" part.  Yeah this needs a lot more information overall... So much that I sort of don't even want to support migration.... I feel like this is a topic of discussion... :/

Reply:
Made a link for each to make it clearer.